### PR TITLE
[ADD] Creating automated actions in Odoo

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: docs/odoo/models/ir.cron.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                # your code
+
+**2. Creating the automated action**
+
+
+    `If you want to build new modules in the guidelines from Odoo you should add the code for an automated action under yourDefaultModule/data/ in a separate XML file.`
+
+An important thing to note with automated actions is that they should always be defined within a noupdate field since this shouldn’t be updated when you update your module.
+
+.. code-block:: shell
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: xml

    .. code-block:: xml

File: docs/odoo/models/ir.cron.rst [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            	<!--<field name="nextcall" ></field>--> 
+            	<field name="model" eval="'model.name '" />
+            	<field name="function" eval="'method_name '" />
+            	<field name="args" eval="" />
+		<!--<field name="priority" eval="5" />--> 
+            </record>
+        </data>
+    </openerp>
+
+The first thing you’ll notice is the data ``noupdate="1"``, this is telling Odoo that all code within this tag shouldn’t be updated when you update your module.
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: No need for future tense:

    you'll

File: docs/odoo/models/ir.cron.rst [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+.. code-block:: shell
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: xml

File: docs/odoo/models/ir.cron.rst [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+    <record id="unique_name" model="ir.cron">
+
+The id is an unique identifier for Odoo to know what record is linked to which id. The model called ("ir.cron") is the model specifically made by Odoo for all automated actions. This model contains all automated actions and should always be specified.
+
+.. code-block:: shell
+
+    <field name="name">Name </field>
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: xml

File: docs/odoo/models/ir.cron.rst [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+    <field name="interval_number">1</field>
+
+Number of times the scheduler is to be called based on the "interval_type" 
+
+.. code-block:: shell
+
+    <field name="interval_type">days</field>
+Interval Unit.
+It should be one value for the list: ``'minutes'``, ``'hours'``, ``'days'``, ``'weeks'``, ``'months'``.
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: Don't use quotes here. Correct format: ``minutes``

    It should be one value for the list: ``'minutes'``, ``'hours'``, ``'days'``, ``'weeks'``, ``'months'``.

File: docs/odoo/models/ir.cron.rst [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+The priority of the job, as an integer: 0 means higher priority, 10 means lower priority.
+
+
+**Defaults.**
+
+
++------------------+-----------------------------------------------------------+
+| Name             | Definition                                                |
++==================+===========================================================+
+| nextcall	   | lambda *a: time.strftime(DEFAULT_SERVER_DATETIME_FORMAT   |
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: quote the code:

     +| nextcall	   | ``lambda *a: time.strftime(DEFAULT_SERVER_DATETIME_FORMAT``   |

File: docs/odoo/models/ir.cron.rst [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
++------------------+-----------------------------------------------------------+
+| priority         | 5                                                         |
++------------------+-----------------------------------------------------------+
+| user_id          | lambda obj,cr,uid,context: uid                            |
```
Comment (yelizariev) [inline docs/odoo/models/ir.cron.rst]: quote the code:

**Comment by GabbasovDinar** [comment]
@yelizariev DONE

**Pull request merged**